ls1intum/Artemis
# Show summary after exam has ended

### Checklist

### Motivation and Context


Students should be able to see the summary after they participated in an exam and the exam is over

### Description

remove check to load student exam only before end date
add call to load studentexam if exam is over

### Steps for Testing


1. Create an Exam (exercises, add students, generate studentexam + prepare exercises)
2. set date so that exam is visible, started and not yet finished -> should work as before
3. set end that so that the exam has ended -> student can see the summary of his/her participation in the exam
